### PR TITLE
[TASK] Add two new ISSUE_TEMPLATES, feature and task

### PR DESCRIPTION
**Pull request by linxpinx**

### CODE REVIEW

**Review by oliverklee — CHANGES_REQUESTED** [review]
Looks generally good to me. I propose we also add a section "Additional information" below.

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -12,3 +12,6 @@ What would you like to achieve with this feature? Maybe in form of a User Story.
 
 ## Acceptance Criteria
 - [ ] 
+
+## Additional Information
+You can add any useful informations here, e.g. links, tipps ect.
```
Comment (oliverklee) [inline .github/ISSUE_TEMPLATE/feature.md:17]:
```suggestion
Anything else that would be helpful to know to implement this feature.
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 2]
```diff
@@ -12,3 +12,6 @@ What would you like to achieve with this task? Maybe in form of a User Story.
 
 ## Acceptance Criteria
 - [ ] 
+
+## Additional Information
+You can add any useful informations here, e.g. links, tipps ect.
```
Comment (oliverklee) [inline .github/ISSUE_TEMPLATE/task.md:17]:
```suggestion
Anything else that would be helpful to know to complete this task.
```

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]
Looks good to me. @DanielSiepmann @lukaszuznanski Could you also please have a look? Thanks!

**Review by lukaszuznanski — APPROVED** [review]

**Pull request merged**